medusa/reconstruct/expand.py

```diff
@@ -1,261 +1,72 @@
-
-    # Cycle through all conditions and gapfill. After each gapfill iteration,
-    # our strategy is to reduce the cost for the reactions returned by the
-    # previous solution to 0, such that they are automatically included in
-    # the model for the next condition.
-    if gapfill_type is "integer":
-        solutions =  _integer_iterative_binary_gapfill(model,
-                              phenotype_dict,
-                              cycle_order,
-                              universal=universal,
-                              output_ensemble_size=output_ensemble_size,
-                              lower_bound=lower_bound,
-                              penalties=penalties,
-                              demand_reactions=demand_reactions,
-                              exchange_reactions=exchange_reactions,
-                              integer_threshold=inclusion_threshold)
-    elif gapfill_type is "continuous":
-        solutions = _continuous_iterative_binary_gapfill(model,
-                              phenotype_dict,
-                              cycle_order,
-                              universal=universal,
-                              output_ensemble_size=output_ensemble_size,
-                              lower_bound=lower_bound,
-                              penalties=penalties,
-                              demand_reactions=demand_reactions,
-                              exchange_reactions=exchange_reactions,
-                              flux_cutoff=inclusion_threshold,
-                              exchange_prefix='EX_')
-
-    ensemble =_build_ensemble_from_gapfill_solutions(model,solutions,
-                                                    universal=universal)
-    return ensemble
-
-def _continuous_iterative_binary_gapfill(model,phenotype_dict,cycle_order,
+def _continuous_iterative_binary_gapfill(model = None, 
                       universal=None, output_ensemble_size=1,
-                      lower_bound=0.05, penalties=None,
+                      lower_bound=0.05, reaction_probability=None,
                       demand_reactions=False,
                       exchange_reactions=False,
                       flux_cutoff=1E-8,
                       exchange_prefix='EX_'):
+    """ 
+    This function takes the gapfilled removed_model, penality, universla model,
+    demand_reaction, exchange reaction, flux_cutff and exchanges_prefix. It returns 
+    the undated a solution with as set of coefficients of the added reactions
+    """
+    
     if exchange_reactions:
         raise NotImplementedError("Inclusion of new exchange reactions is not"
                             "supported for continuous gapfill")
     if demand_reactions:
         raise NotImplementedError("Inclusion of demand reactions is not"
                             "supported for continuous gapfill")
-
     solutions = []
     gapfiller = universal.copy()
-
-
-
+    model.reactions.EX_cpd00007_e.lower_bound = 0
```

Comment:
For the Medusa codebase, we should let the user specify the media conditions rather than changing them like this (so the user would close this oxygen exchange reaction in their model _before_ trying to call the Medusa function to gapfill it and generate an ensemble)